backend/tarsy/repositories/history_repository.py

```
@@ -660,39 +296,730 @@ def update_session_current_stage(
+        self,
+        status: Optional[Union[str, List[str]]] = None,
+        agent_type: Optional[str] = None,
+        alert_type: Optional[str] = None,
+        search: Optional[str] = None,
+        start_date_us: Optional[int] = None,
+        end_date_us: Optional[int] = None,
+        page: int = 1,
+        page_size: int = 20
+    ) -> PaginatedSessions:
+        """
+        Internal method returning PaginatedSessions model - builds type-safe models DIRECTLY from database.
```

Ruff (0.12.2)

Line too long (107 > 88)

(E501)